README.md

```diff
@@ -145,6 +145,23 @@ seconds past the minute).
 A big shout out to [rufus-scheduler](http://github.com/jmettraux/rufus-scheduler)
 for handling the heavy lifting of the actual scheduling engine.
 
+### Enabling/Disabling jobs
+
+By default all the jobs are enabled. There is an option at job's definition to change
```

Comment:
'By default all the jobs are enabled. There is an option at job's configuration to change'